README.md

```diff
@@ -211,6 +215,10 @@ The bold notation of key (such as **Elements**) means that it's a necessary key.
 | **md_nstep** | Integer | 10 | Steps of AIMD in stage 3. If it's not equal to settings via `NSW` in `md_incar`, DP-GEN will follow `NSW`.
 | **coll_ndata** | Integer | 5000 | Maximal number of collected data.
 | type_map | List | [ "Mg", "Al"] | The indices of elements in deepmd formats will be set in this order.
+| init_fp_style | String | "ABACUS" or "VASP" | First-principle software. If this key is abscent, the default value will be "VASP".
+| relax_kpt | String | "....../KPT" | Path of `KPT` file for relaxation in stage 1. Only useful if `init_fp_style` is "ABACUS".
+| md_kpt | String | "....../KPT" | Path of `KPT` file for MD simulations in stage 3. Only useful if `init_fp_style` is "ABACUS".
```

Comment:
It is not good idea to explicitly set the number of kpoint, because the k-mesh should be adjusted according to the size of the system. For example different `super_cell` settings needs different number of kpoints. 

A better and recommended way of setting kmesh is to set kspacing. you can check `auto_test.lib.vasp.make_kspacing_kpoints` to make a kmesh from a given kspacing.

Reply:
Sorry for the delay. I did not receive the notifications orz. I will solve the two issues as suggested.

Reply:
I have added `relax_kspacing` and `md_kspacing` in `param.json` for dpgen to generate the `KPT` files. I also reserved the previous `relax_kpt` and `md_kpt` parameters. Both methods are available.